### PR TITLE
Show bing disclaimer only for Bing APIs, Improve error message when a…

### PR DESCRIPTION
**Pull request by yangyuan**
…ccount creation failed.

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: src/CognitiveServices/CognitiveServices/CognitiveServicesAccount/NewAzureCognitiveServicesAccount.cs [hunk 1]
```diff
@@ -118,23 +119,36 @@ public override void ExecuteCmdlet()
...
+                        CognitiveServicesAccount createAccountResponse = CognitiveServicesClient.Accounts.Create(
+                                        ResourceGroupName,
+                                        Name,
+                                        createParameters);
+                    }
+                    catch (Exception ex)
+                    {
+                        // Give users a specific message says `Failed to create Cognitive Services account.`
+                        // Details should able be found in the exception.
+                        WriteError(new ErrorRecord(new Exception("Failed to create Cognitive Services account.", ex), string.Empty, ErrorCategory.NotSpecified, null));
```
Comment (cormacpayne) [inline src/CognitiveServices/CognitiveServices/CognitiveServicesAccount/NewAzureCognitiveServicesAccount.cs]: @yangyuan any reason that you aren't just throwing the new `Exception` here?

Comment (yangyuan) [reply src/CognitiveServices/CognitiveServices/CognitiveServicesAccount/NewAzureCognitiveServicesAccount.cs]: No specific reason. In here, `ErrorRecord` doesn't contribute extra information, so it's just for consistency.
Do you recommend to change to simple `Exception`?

Comment (cormacpayne) [reply src/CognitiveServices/CognitiveServices/CognitiveServicesAccount/NewAzureCognitiveServicesAccount.cs]: @yangyuan we advise just throwing an `Exception` in this case

**Comment by maddieclayton** [comment]
@yangyuan Ping on comment above.

**Comment by maddieclayton** [comment]
Closing for inactivity - please reopen when you are ready to address comments above.

**Comment by yangyuan** [comment]
@maddieclayton sorry for the delay, I've updated the PR to just send an Exception

File: src/CognitiveServices/CognitiveServices.sln [hunk 2]
```diff
@@ -7,6 +7,12 @@ EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Tests", "Tests", "{13BB1C60-A70E-4780-ABEF-D5EE36754085}"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "CognitiveServices.Test", "CognitiveServices.Test\CognitiveServices.Test.csproj", "{F6F8A360-F203-4161-98A7-60097AB94193}"
+	ProjectSection(ProjectDependencies) = postProject
```
Comment (niander) [inline src/CognitiveServices/CognitiveServices.sln]: Do you need these entries in the solution file? If you are not sure why they were added could you remove and test that everything is ok?

Comment (yangyuan) [reply src/CognitiveServices/CognitiveServices.sln]: Its not needed (auto added by VS)

**Pull request merged**